x/concentrated-liquidity/incentives.go

```diff
@@ -344,17 +344,17 @@ func (k Keeper) updateUptimeAccumulatorsToNow(ctx sdk.Context, poolId uint64) er
 		curUptimeDuration := types.SupportedUptimes[uptimeIndex]
 
 		// Qualifying liquidity is the amount of liquidity that satisfies uptime requirements
-		qualifyingLiquidity, err := uptimeAccum.GetTotalShares()
+		uptimeAccumTotalShares, err := uptimeAccum.GetTotalShares()
```

Comment:
nit: I also prefer the old name here but not feeling strongly

Reply:
maybe it's because I lack context on incentives design, why is it called "qualified liquidity"?

Reply:
I actually also prefer the old name

Reply:
changing back to old name